src/services/treatment.service.ts

```diff
@@ -17,9 +18,21 @@ export default class TreatmentService {
     private readonly treatmentRepository: Repository<Treatment>,
   ) {}
 
-  async getAllTreatments(): Promise<Treatment[]> {
+  async get(param: string): Promise<Treatment[]> {
     try {
-      return this.treatmentRepository.find();
+      const treatment = await this.treatmentRepository.find({
```

Comment:
```suggestion
      const treatments = await this.treatmentRepository.find({
```

this is fetching several treatments, right?